services/api/src/messaging/players.service.ts

```diff
@@ -232,6 +299,42 @@ export const subscribe = (ws: GameWebSocket, newPlayer: Player): void => {
 
 export const unsubscribe = (ws: GameWebSocket): void => {
   const players = getPlayers(ws.roomId);
+  try {
+    const playerId = findPlayerByConnection(ws).id;
+
+    if (
+      gameService.getVoters(ws.roomId).length === 2 &&
```

Comment:
why is this condition needed?

Reply:
Because, when there is only to people in the room and the game is not finished, when one of them logs out, in my opinion the game state should become game not started (we don't allow player to make votes, when he's the only one in the room).